### DISCUSSION TITLE
TheHive

### DISCUSSION THREAD

**Discussion by nimmy-shakya** [Security-Onion-Solutions/securityonion#8817] [Unsupported Versions] [answered]
How to integrate TheHive in security onion version 2.3.0 ?

**Comment by vanh20** [comment]
Hello nimmy-shakya,

   I believe this was already answered recently [here](https://github.com/Security-Onion-Solutions/securityonion/discussions/8781). Please see the [suggested answer](https://docs.securityonion.net/en/2.3/soc-customization.html#escalation) by dougburks.

**Comment by dougburks** [comment, marked as answer]
@nimmy-shakya As we responded to you on [Twitter](https://twitter.com/securityonion/status/1574106476829970432):

We no longer include or support TheHive.

We recommend using our Cases interface:
https://docs.securityonion.net/en/2.3/cases.html

There is an option to escalate to TheHive:
https://docs.securityonion.net/en/2.3/soc-customization.html#escalation

**Reply by phantomanalytical** [reply]
I just realized this today when I escalated a case on my home instance. I'm sure there are reasons for no longer including it, but I have to say that is pretty upsetting. I absolutely love TheHive and Cortex. Does the current Cases interface offer the same ability to automatically query IOCs via API to different analyzers? If not that is a major setback for me.

**Reply by dougburks** [reply]
@phantomanalytical 
> I just realized this today when I escalated a case on my home instance. I'm sure there are reasons for no longer including it, but I have to say that is pretty upsetting. I absolutely love TheHive and Cortex.

From https://blog.securityonion.net/2022/01/coming-soon-in-security-onion-23100.html:
_In September, 2021, StrangeBee announced a change to TheHive's licensing model and ended support for TheHive version 3 effective December 31, 2021:
https://medium.com/strangebee-announcements/faq-for-thehive-5s-upcoming-distribution-model-af0ccb95d18
The new licensing model for TheHive version 5 is not compatible with our project so we must say goodbye to TheHive and Cortex._

>  Does the current Cases interface offer the same ability to automatically query IOCs via API to different analyzers? If not that is a major setback for me.

Yes, we added our own analyzers in June:
https://blog.securityonion.net/2022/06/security-onion-23130-now-available.html

For more information about Cases and Analyzers, please see:
https://docs.securityonion.net/en/2.3/cases.html

If you have further questions or problems, please start a new discussion rather than replying to somebody else's discussion. Thanks!
